### ISSUE TITLE
`o-icon-uri.icon` span missing role

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-modules/ValueSuggest#88]
`o-icon-uri.icon` icons within item/item set edit fields are in a span with an `aria-label` of "URI". This `aria-label` is not currently detected by assistive tech as the span is missing a role.
